nodeinstaller/node-installer.go

```diff
@@ -204,12 +213,17 @@ func containerdRuntimeConfig(basePath, configPath string, platform platforms.Pla
 	return os.WriteFile(configPath, rawConfig, os.ModePerm)
 }
 
-func patchContainerdConfig(runtimeName, basePath, configPath string, platform platforms.Platform) error {
+func patchContainerdConfig(basePath, configPath string, platform platforms.Platform) error {
 	existingRaw, existing, err := parseExistingContainerdConfig(configPath)
 	if err != nil {
 		existing = constants.ContainerdBaseConfig()
 	}
 
+	runtimeName, err := manifest.RuntimeHandler(platform)
+	if err != nil {
+		return fmt.Errorf("getting runtime name: %w", err)
+	}
+
```

Comment:
I think it is fine to keep on taking the runtime handler as input here. We shouldn't start to move calls to such global magic all the way down but rather try to call it somewhere top and pass the value down. This will also keep our test stable and independent from how the runtime handler embedding works, the change of test values in this PR is unnecessary.